composer.json

```diff
@@ -8,6 +8,7 @@
         "symfony/monolog-bundle": "^3.0",
         "easycorp/easy-log-handler": "dev-master",
         "symfony/web-profiler-bundle": "^3.3",
+        "symfony/twig-bundle": "^3.3",
```

Comment:
yeah, this is the alternative. And it may be better by avoiding to duplicate the knowledge about what the profiler pack needs